gradle/instrumentation.gradle

```diff
@@ -13,6 +13,10 @@ byteBuddy {
 
 apply from: "${rootDir}/gradle/java.gradle"
 
+tasks.withType(Test) {
+  forkEvery = 1
+}
+
```

Comment:
Is this still needed if we call `addByteBuddyRawSetting` early enough?

Reply:
unfortunately, ya, net.bytebuddy.raw solved the `Cannot resolve bounds of unresolved type variable`, but didn't solve the `Cannot resolve type description` that i tried to resolve initially in #349, and ended up i think with better solution in this PR without having to ignore any failures